source/index.rst

```diff
@@ -0,0 +1,39 @@
+Revspeech API Documentation
+===========================
+
+This Revspeech API aims to deliver quality speech-text recognition via an API. All public methods and 
+objects are documented here for developer reference.
+
+*This documentation is a work in progress and subject to change*
+
+Base Endpoint
+**************
+
+The base url for this version of the api is
+
+ ``https://api.rev.com/revspeech/v1beta``
+
+All endpoints described in this documentation are relative to this base url
+
+.. toctree::
+    :maxdepth: 2
+    :caption: Authorization:
+
+    authorization/api-key
```

Comment:
I would just put the authorization info right here. its a short section, no need to make them click further down